h/util/metrics.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import unicode_literals
+import newrelic.agent
+
+
+def record_search_query_param_usage(params, separate_replies, record_param=newrelic.agent.add_custom_parameter):
```

Comment:
We discussed this in a Slack call: let's remove the `record_param` param here—external callers shouldn't be able to dictate what recording mechanism happens here and it'll simplify this module if it just does whatever it needs to internally.